Tests/WorkspaceTests/WorkspaceTests.swift

```diff
@@ -1627,7 +1627,8 @@ final class WorkspaceTests: XCTestCase {
                 try localFileSystem.createDirectory(root)
             }
 
-            try localFileSystem.writeFileContents(roots[2].appending(components: Manifest.filename)) { stream in
+            let manifestPath = Manifest.versionSpecificManifest(path: roots[2], fileSystem: localFileSystem)
+            try localFileSystem.writeFileContents(manifestPath) { stream in
                 stream <<< "import PackageDescription" <<< "\n"
                 stream <<< "let package = Package(name: \"root0\")"
             }
```

Comment:
We should add a test case that we actually load the correct tools version when version specific manifest is present.